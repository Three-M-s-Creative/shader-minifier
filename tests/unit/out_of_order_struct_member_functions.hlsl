struct MyStruct {
    float my_function() {
        return my_other_function() + 5;
    }
    
    float my_other_function() {
        return 10 + member_var;
    }

    int member_var;
};

float my_other_function() {
    return 10 + member_var;
}