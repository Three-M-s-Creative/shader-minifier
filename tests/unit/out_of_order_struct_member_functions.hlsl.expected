struct Q{
  int t;
  float P()
  {
    return O()+5;
  };
  float O()
  {
    return 10+member_var;
  };
};
